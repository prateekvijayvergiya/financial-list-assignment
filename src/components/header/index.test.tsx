import Component from './'
import makeSnapshot from '../../common/testing/makeSnapshot'

jest.mock('./headerButton', () => 'HeaderButton')

describe('TableHeader', () => {
  it('renders correctly', () => {
    const props = {
      handleSortChange: jest.fn(),
      sortConfig: null
    }
    makeSnapshot(<Component {...props}/>)
  })
})
