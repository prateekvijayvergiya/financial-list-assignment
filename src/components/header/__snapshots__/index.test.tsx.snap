// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader renders correctly 1`] = `
<thead>
  <tr>
    <HeaderButton
      handleSortChange={[MockFunction]}
      sortConfig={null}
      sortKey="assetClass"
      title="Asset Class"
    />
    <HeaderButton
      handleSortChange={[MockFunction]}
      sortConfig={null}
      sortKey="price"
      title="Price"
    />
    <HeaderButton
      handleSortChange={[MockFunction]}
      sortConfig={null}
      sortKey="ticker"
      title="Ticker"
    />
  </tr>
</thead>
`;
